.github/workflows/flake8.yml

```diff
@@ -0,0 +1,226 @@
+name: Python code quality check
+
+on:
+- push
+- pull_request
+
+jobs:
+
+  flake8-grass7-db-scripts:
+
+    runs-on: ubuntu-18.04
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.8
+    - name: Install
+      run: |
+        python -m pip install --upgrade pip
+        pip install flake8==3.8.0
+    - name: Run Flake8
+      run: |
+        cd grass7/db
+        flake8 --config=../.flake8 --count --statistics --show-source --jobs=$(nproc) .
+
+
+  flake8-grass7-display-scripts:
+
+    runs-on: ubuntu-18.04
```

Comment:
I would just go with 20.04 here. (It is now matured as ubuntu-latest in GH Action and so I would use it by default unless there is a reason to use something else.)

Reply:
Sure fine with 20.04.